test/gtest/uct/test_uct_perf.cc

```diff
@@ -15,8 +15,64 @@ extern "C" {
 #define MB                        pow(1024, -2)
 #define UCT_PERF_TEST_MULTIPLIER  5
 #define UCT_ARM_PERF_TEST_MULTIPLIER  15
+#define UCT_CUDA_PERF_TEST_MULTIPLIER 10
 
 class test_uct_perf : public uct_test, public test_perf {
+public:
+    void test_execute(unsigned flags = 0) {
```

Comment:
minor: this function is pretty big - maybe better define its body outside the class definition?